### DISCUSSION TITLE
grpcio version not updated for pymilvus 2.0.2

### DISCUSSION THREAD

**Discussion by chaitaliprabhu** [milvus-io/milvus#16557] [Q&A and General discussion]
Should there be an update for the grpcio version used in pymilvus 2.0.2? 
It currently uses grpcio 1.37.1 and 'Building wheel for grpcio (setup.py)' throws an error.
The latest version of grpcio is 1.44.0.

Error on _pip install pymilvus_

error: legacy-install-failure

× Encountered error while trying to install package.
╰─> grpcio

**Comment by yhmo** [comment]
Last year, we found the latest grpcio lib cannot compatible with milvus: https://github.com/milvus-io/pymilvus/issues/517
1.37.0 can work but 1.38.0 has segment fault.

Seems the error you encountered is related to your system environment. I search "legacy-install-failure" by google, maybe this link can help: https://trustsu.com/python/error-legacy-install-failure/
